types/coin.go

```diff
@@ -368,6 +368,23 @@ func (coins Coins) IsAllLTE(coinsB Coins) bool {
 	return coinsB.IsAllGTE(coins)
 }
 
+// IsAnyGT returns true iff for any denom in coins, the denom is present at a
+// greater amount in coinsB.
+func (coins Coins) IsAnyGT(coinsB Coins) bool {
+	if len(coinsB) == 0 {
+		return false
+	}
+
+	for _, coin := range coins {
+		amt := coinsB.AmountOf(coin.Denom)
+		if coin.Amount.GT(amt) && !amt.IsZero() {
```

Comment:
Not sure about `!amt.IsZero()`. If `setA` contains a coin `Denom=X` with `Amount=1` and `setB` does not contain the denom `X`, I guess this should be interpreted as `Denom=X in setB has Amount=0`

Reply:
Well, per the godoc:

```
// IsAnyGT returns true iff for any denom in coins, the denom is present at a
// greater amount in coinsB.
```

It is important to be consistent across all of these methods. This operates the same way as `IsAnyGTE`.

Reply:
It makes sense. May I suggest to enrich the documentation with some examples? We treat absence of denom in a coin set as 0 amount-denom in many cases across the code base.

Reply:
True. Sure thing!